src/components/TeacherViews/LessonShare/LessonShare.view.tsx

```diff
@@ -47,12 +51,18 @@ type LessonShareProps =
   | {
       isCanonical: false;
       lesson: LessonPathway & {
+        isSpecialist: false;
+        developmentStageTitle?: string | null;
```

Comment:
this will never be present for non specialist right?

Reply:
No never present, but the TS checker wouldn't let the isSpecialist check work kept throwing up a type error